specs/altair/beacon-chain.md

```diff
@@ -621,11 +621,14 @@ def process_justification_and_finalization(state: BeaconState) -> None:
 ```python
 def process_inactivity_updates(state: BeaconState) -> None:
     for index in get_eligible_validator_indices(state):
+        # Increase inactivity score of inactive validators
         if index in get_unslashed_participating_indices(state, TIMELY_TARGET_FLAG_INDEX, get_previous_epoch(state)):
-            if state.inactivity_scores[index] > 0:
-                state.inactivity_scores[index] -= 1
-        elif is_in_inactivity_leak(state):
+            state.inactivity_scores[index] -= min(1, state.inactivity_scores[index])
```

Comment:
This line logic unchanged but removes the conditional in favor of this `min` condition

Reply:
Yep! I The min approach is better for this case because it more easily generalizes to subtracting values other than 1.